src/nls/fi/strings.js

```diff
@@ -418,6 +419,7 @@ define({
     "CMD_SHOW_IN_EXPLORER"                : "Näytä Resurssienhallinnassa",
     "CMD_SHOW_IN_FINDER"                  : "Näytä Finderissä",
     "CMD_SHOW_IN_OS"                      : "Näytä käyttöjärjestelmässä",
+    "CMD_SWITCH_PANE_FOCUS"               : "Vaihda aktiivinen paneeli",
```

Comment:
I think *Vaihda ruudun valinta* would be better. (*Pane* has been translated to *ruutu* elsewhere.)